### ISSUE TITLE
Impossible to reach reported accuracy

### ISSUE THREAD

**Issue by Nadni** [JakeKandell/NBA-Predict#5]
Hi,
I have tried to re-train the model by running createModel(), though the accuracies I got in 6 different trials are as following:
0.671957672
0.662698413
0.650793651
0.636243386
0.624338624
0.657407407
How did you manage to get the reported accuracy of 70%?

**Comment by JakeKandell** [comment]
Well when you use createModel(), you are generating a completely new model trained on a different set of games with different weights. Thus, the accuracy will differ. I honestly generated about 75 total models in order to settle on the best one.

**Comment by Nadni** [comment]
In this case I think the accuracy should be reported both on a validation set (the one you check to choose the best model out of the 75) and on another test set, to avoid a bias in the estimation of accuracy.

**Comment by JakeKandell** [comment]
That's definitely a good idea. I just went with the idea that the sample size was large enough to minimize that problem and scraping a whole new set of games with data takes a somewhat long time.
